python/ml4ir/applications/ranking/tests/test_feature_processing.py

```diff
@@ -0,0 +1,31 @@
+import string
+
+from ml4ir.applications.ranking.tests.test_base import RankingTestBase
+from ml4ir.base.features import preprocessing
+
+
+class RankingModelTest(RankingTestBase):
```

Comment:
Why should this be in rankin module, when we test the base package?
Do you plan to have all the tests ins /ranking?